### PR TITLE
Added Ctrl+F shortcut for opening the search box

### LINKED ISSUE #4326: Shortcut for search button
**Is your feature request related to a problem?**

- I would like to use the search directly using the keyboard.
- I would like to type the path myself without using the mouse.

**Describe the solution**

We need 2 shortcuts. One for focusing on the "path bar" (idk if that's its name) and the other one is to activate search button.

- CTRL + F should work for the search.
- Alt + F (? even better, make it configurable in the settings) for focusing to the path bar.

**Describe alternatives you've considered**

Alternatively or additionally, you can make the app focus on the "path bar" when we open a new tab. This is how most browsers behave. We open a new tab using CTRL+T and the focus is on the search bar.

### PR DESCRIPTION
**Pull request by Jaiganeshkumaran**
Fixes #4326 

For #2447. Note: currently pressing Esc in the search box focuses on the next element in the visual tree instead of the element that was previous selected

### CODE REVIEW

**Review by gave92 — APPROVED** [review]

**Review by duke7553 — DISMISSED** [review]

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -508,7 +508,7 @@
     <value>Refresh (F5)</value>
   </data>
   <data name="NavSearchButton.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Search</value>
+    <value>Search (Ctrl+F)</value>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:511]:
```suggestion
    <value>Search (Ctrl + F)</value>
```
Other shortcuts have a space before and after the `+` character. Perhaps we can change that to be consistent with other apps but that can be done in a different pull request.

File: Files/UserControls/NavigationToolbar.xaml [hunk 2]
```diff
@@ -864,7 +864,11 @@
                 FontSize="16"
                 IsEnabled="{x:Bind IsPageTypeNotHome, Mode=OneWay}"
                 Style="{StaticResource ToolBarButtonStyle}"
-                ToolTipService.ToolTip="Search" />
+                ToolTipService.ToolTip="Search (Ctrl+F)">
```
Comment (yaichenbaum) [inline Files/UserControls/NavigationToolbar.xaml:867]:
```suggestion
                ToolTipService.ToolTip="Search (Ctrl + F)">
```

File: Files/UserControls/NavigationToolbar.xaml [hunk 3]
```diff
@@ -864,7 +864,11 @@
...
+                <Button.KeyboardAccelerators>
+                    <KeyboardAccelerator Key="F" Modifiers="Control" />
```
Comment (yaichenbaum) [inline Files/UserControls/NavigationToolbar.xaml:869]: Can you move the keyboard accelerator to ModernShellPage to be consistent with other shortcuts?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/UserControls/NavigationToolbar.xaml (hunk 3 above)
Comment (lukeblevins) [reply Files/UserControls/NavigationToolbar.xaml]: @yair100 It doesn't really matter here because this is registered in XAML. In other words, it makes sense for that control to have its own shortcut.

**Review by duke7553 — DISMISSED** [review]

File: Files/UserControls/NavigationToolbar.xaml (hunk 3 above)
Comment (yaichenbaum) [reply Files/UserControls/NavigationToolbar.xaml:869]: @duke7553 Unfortunately this causes issues with the shortcut triggering some of the time, this was one of the reasons we moved a lot of the shortcuts to ModernShellPage.

File: README.md [hunk 4]
```diff
@@ -28,3 +28,5 @@ In the short term, we plan to focus on crucial functionality to achieve parity w
 
 ## FAQ
 Have any questions? Check out our documentation [site](https://files-community.github.io/docs/#/)!
+
+
```
Comment (yaichenbaum) [inline README.md:32]:
```suggestion
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -92,3 +92,5 @@ _Revision 2.2. Posted 4 February 2016._
 _Revision 2.1. Posted 23 June 2014._
 
 _Revision 2.0, adopted by the [Stumptown Syndicate](http://stumptownsyndicate.org) board on 10 January 2013. Posted 17 March 2013._
+
+
```
Comment (yaichenbaum) [inline CODE_OF_CONDUCT.md:96]:
```suggestion
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]
Can you resolve the merge conflicts?

**Comment by d2dyno1** [comment]
@Jaiganeshkumaran Can you report progress on this?

File: Files/Views/ModernShellPage.xaml [hunk 6]
```diff
@@ -196,15 +201,105 @@
...
+                </Custom:GridSplitter>
+
+                <controls:PreviewPane
+                    x:Name="PreviewPane"
+                    x:Uid="SelectedFilePreviewPane"
+                    Grid.Row="1"
+                    Grid.Column="2"
+                    x:Load="{x:Bind PreviewPaneEnabled, Mode=OneWay}"
+                    AutomationProperties.Name="Selected file preview pane"
+                    SelectedItems="{x:Bind ContentPage.SelectedItems, Mode=OneWay}" />
```
Comment (yaichenbaum) [inline Files/Views/ModernShellPage.xaml:302]: @Jaiganeshkumaran Can you fix the changes you made here?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran Can you solve the merge conflicts?

**Comment by Jaiganeshkumaran** [comment]
> @Jaiganeshkumaran Can you solve the merge conflicts?

Done!

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran It looks like there are still some issues, with the exception of the en-us resource file, can you revert the changes you made to the other resource files?

File: Files/Strings/ja-JP/Resources.resw [hunk 7]
```diff
@@ -340,10 +340,10 @@
...
   <data name="LightTheme" xml:space="preserve">
-    <value>ライト</value>
+    <value>明るい</value>
   </data>
   <data name="DarkTheme" xml:space="preserve">
-    <value>ダーク</value>
+    <value>暗い</value>
   </data>
   <data name="ApplicationTimeStye" xml:space="preserve">
     <value>アプリ</value>
```
Comment (yaichenbaum) [inline Files/Strings/ja-JP/Resources.resw:349]: Can you revert the changes to this file?

File: Files/Strings/hu-HU/Resources.resw [hunk 8]
```diff
@@ -1732,19 +1723,13 @@
...
-  </data>
-  <data name="SettingsUsePreemptiveCache.Header" xml:space="preserve">
-    <value>Előzetes gyorsítótárazás (almappák beolvasása navigáláskor)</value>
-  </data>
   <data name="BundlesWidgetMoreOptionsButton.AutomationProperties.Name" xml:space="preserve">
-    <value>További munkaterület lehetőségek</value>
+    <value>További munkafelület lehetőségek</value>
   </data>
   <data name="DrivesWidgetMoreOptionsButton.AutomationProperties.Name" xml:space="preserve">
     <value>További meghajtó lehetőségek</value>
```
Comment (yaichenbaum) [inline Files/Strings/hu-HU/Resources.resw:1735]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 9]
```diff
@@ -2450,6 +2451,14 @@
...
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
+          <target state="new">Show Library section on the sidebar</target>
+        </trans-unit>
         <trans-unit id="PinItemToStart.Text" translate="yes" xml:space="preserve">
           <source>Pin to the Start Menu</source>
           <target state="new">Pin to the Start Menu</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2464]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.zh-Hans.xlf [hunk 10]
```diff
@@ -2447,6 +2448,14 @@
...
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
+          <target state="new">Show Library section on the sidebar</target>
+        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.zh-Hans.xlf:2458]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.uk-UA.xlf [hunk 11]
```diff
@@ -2447,6 +2448,15 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.uk-UA.xlf:2455]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.tr-TR.xlf [hunk 12]
```diff
@@ -2436,6 +2436,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.tr-TR.xlf:2441]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.ta.xlf [hunk 13]
```diff
@@ -2449,6 +2450,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ta.xlf:2458]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.sv-SE.xlf [hunk 14]
```diff
@@ -2447,6 +2448,14 @@
...
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
+          <target state="new">Show Library section on the sidebar</target>
+        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.sv-SE.xlf:2458]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.ru-RU.xlf [hunk 15]
```diff
@@ -423,8 +423,9 @@
           <target state="translated">Обновить (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated">Поиск</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation">Поиск</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ru-RU.xlf:428]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.pt-BR.xlf [hunk 16]
```diff
@@ -491,8 +491,9 @@
...
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated">Pesquisar</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation">Pesquisar</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
         </trans-unit>
         <trans-unit id="NavMoreButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>More options</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.pt-BR.xlf:499]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.pl-PL.xlf [hunk 17]
```diff
@@ -2449,6 +2450,14 @@
           <source>Files is running as administrator</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.pl-PL.xlf:2450]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.or-IN.xlf [hunk 18]
```diff
@@ -504,8 +504,9 @@
           <target state="translated">ସତେଜ କର (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated">ସନ୍ଧାନ କର</target>
+          <source>Search (Ctrl + F)</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.or-IN.xlf:507]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.nl-NL.xlf [hunk 19]
```diff
@@ -2450,6 +2451,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.nl-NL.xlf:2455]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.ko-KR.xlf [hunk 20]
```diff
@@ -419,8 +419,9 @@
...
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated">검색</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation">검색</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
         </trans-unit>
         <trans-unit id="NavMoreButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>More options</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ko-KR.xlf:427]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.ja-JP.xlf [hunk 21]
```diff
@@ -2442,6 +2443,14 @@
           <source>Unpin from the Start Menu</source>
           <target state="translated">スタートメニューから固定解除</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
+          <target state="new">Show Library section on the sidebar</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ja-JP.xlf:2452]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.it-IT.xlf [hunk 22]
```diff
@@ -2442,6 +2443,14 @@
           <source>Unpin from the Start Menu</source>
           <target state="translated">Rimuovi dal Menu Start</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.it-IT.xlf:2446]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 23]
```diff
@@ -2455,6 +2459,14 @@
           <source>Unpin from the Start Menu</source>
           <target state="translated">Levétel a Start menüből</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.hu-HU.xlf:2466]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.hi-IN.xlf [hunk 24]
```diff
@@ -504,8 +504,9 @@
           <target state="translated">ताज़ाकरण (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated">खोजें</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation">खोजें</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.hi-IN.xlf:509]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.he-IL.xlf [hunk 25]
```diff
@@ -495,8 +495,9 @@
           <target state="translated" state-qualifier="tm-suggestion">Refresh (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated" state-qualifier="tm-suggestion">חיפוש</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">חיפוש</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.he-IL.xlf:500]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 26]
```diff
@@ -448,8 +448,9 @@
           <target state="translated" state-qualifier="tm-suggestion">Actualiser (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated" state-qualifier="tm-suggestion">Rechercher</target>
+          <source>Search (Ctrl + F)</source>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">Rechercher</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.fr-FR.xlf:452]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.es-ES.xlf [hunk 27]
```diff
@@ -2442,6 +2443,14 @@
           <source>Unpin from the Start Menu</source>
           <target state="translated">Desanclar de Inicio</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.es-ES.xlf:2449]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 28]
```diff
@@ -423,8 +423,9 @@
           <target state="translated">Aktualisieren (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.de-DE.xlf:425]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.da.xlf [hunk 29]
```diff
@@ -2444,6 +2445,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.da.xlf:2447]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.da-DK.xlf [hunk 30]
```diff
@@ -2444,6 +2445,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.da-DK.xlf:2453]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.cs-CZ.xlf [hunk 31]
```diff
@@ -2464,6 +2465,14 @@
           <source>Files is running as administrator</source>
           <target state="new">Files is running as administrator</target>
         </trans-unit>
+        <trans-unit id="BundlesOpenInNewPane" translate="yes" xml:space="preserve">
+          <source>Open in new Pane</source>
+          <target state="new">Open in new Pane</target>
+        </trans-unit>
+        <trans-unit id="SettingsPreferencesShowLibrarySection.Header" translate="yes" xml:space="preserve">
+          <source>Show Library section on the sidebar</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.cs-CZ.xlf:2473]: Can you revert the changes to this file?

File: Files/MultilingualResources/Files.ar.xlf [hunk 32]
```diff
@@ -528,8 +528,9 @@
           <target state="translated" state-qualifier="tm-suggestion">تحديث (F5)</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Search</source>
-          <target state="translated" state-qualifier="tm-suggestion">بحث</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ar.xlf:532]: Can you revert the changes to this file?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]
